### DISCUSSION TITLE
Alternate name for "Emulator Support" on main docs page

### DISCUSSION THREAD

**Discussion by suspect15** [RetroAchievements/docs#41] [Ideas]
There is quite a bit more information in section of the docs to which "Emulator Support" links.  I think something like "General Information" would be a better catch all, but I am concerned that it would lose the clear indication that emulator support guidance is contained within.  My suspicion is emulator support is the most important doc new RA users would need and that it should likely be a candidate for a very easy to find link.

**Comment by wescopeland** [comment]
What you're saying is valid. Based on the search traffic I've analyzed, I think we should leave emulator support as-is. Google is pumping a lot of juice into that page from it existing for years and years and being considered as "relevant content".

I am wondering if the use case you're describing is https://github.com/RetroAchievements/docs/issues/10.
Essentially: a quickstart page. Based on the traffic reports I've analyzed, I strongly suspect users are confused on how to get started with RetroAchievements. Popular search terms and where users end up from those terms seems to suggest this. 

We should have a page that takes them from (A) having no account whatsoever, to (B) earning their first achievement on a popular game, as fast as humanly possible. Ideally visual in nature, with whatever screenshots are necessary, supplemented by text content.
